test/sheet/drink_sheet.py

```diff
@@ -0,0 +1,36 @@
+from unittest import TestCase
+from unittest.mock import MagicMock
+
+from file import MemoryFile
+from file.sheet import DrinkSheet
+from user.drinks import DrinkFactory
+from user.drinks.types import Coffee
+
+
+class TestDrinkSheet(TestCase):
+    def test_constructor(self):
+        coffee = Coffee(id_number=1, name='Coffee', cost=1.0)
+        drink_factory = DrinkFactory
+
+        mocked_get = MagicMock(return_value=coffee)
+        drink_factory.get = mocked_get
+
+        data = {'id_number': 1, 'name': 'Coffee', 'type': 1, 'cost': 1.0}
+        sheet = DrinkSheet([data])
+
+        mocked_get.assert_called_with(data)
+        self.assertEqual(sheet.data, [coffee])
+
+    def test_to_dict(self):
+        data = {'id_number': 1, 'name': 'Coffee', 'type': 1, 'cost': 1.0}
```

Comment:
```suggestion
         data = {'id_number': 1, 
         'name': 'Coffee', 
         'type': 1, 
         'cost': 1.0}
```
Again here i would have done the same as mentioned above

Reply:
As above, this is fine